packages/mermaid/src/diagrams/flowchart/flowDb.ts

```diff
@@ -264,14 +251,12 @@ export const setDirection = function (dir) {
 /**
  * Called by parser when a special node is found, e.g. a clickable element.
  *
- * @param ids Comma separated list of ids
- * @param className Class to add
+ * @param ids - Comma separated list of ids
+ * @param className - Class to add
  */
-export const setClass = function (ids, className) {
+export const setClass = function (ids: string, className: string) {
   ids.split(',').forEach(function (_id) {
-    // let id = version === 'gen-2' ? lookUpDomId(_id) : _id;
-    let id = _id;
-    // if (_id[0].match(/\d/)) id = MERMAID_DOM_ID_PREFIX + id;
+    const id = _id;
```

Comment:
What about just the following, so we don't need the `const id = _id;`:

```suggestion
  ids.split(',').forEach(function (id) {
```